utils/model_utils.py

```diff
@@ -30,6 +28,18 @@
 	"VisionTransformer": "vision_transformer",
 }
 
+MODEL_INFO = {
+	"resnet18": {
+		"image_size": (256, 256)
+	},
+	"inception_v3": {
+		"image_size": (299, 299)
+	},
+	"vision_transformer": {
+		"image_size": (256, 256)
+	}
+}
```

Comment:
```suggestion
	"vision_transformer": {
		"image_size": (256, 256)
	},
	"bag_of_words": {
		"image_size": (256, 256)
	}
}
```

Reply:
Bag of words doesn't really need image resizing, but to conform to other classifiers and not overcomplicate data augmentation in train.py by having multiple augmentations depending on model architecture, we could add this here